ethers-core/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/gakonst/ethers-rs"
 keywords = ["ethereum", "web3", "celo", "ethers"]
 
 [dependencies]
+fastrlp = { git = "https://github.com/vorot93/fastrlp", features = ["std", "derive", "ethereum-types"] }
```

Comment:
Does this have a release?

Reply:
No, not yet. `0.1.1` does not have the changes, a new release needs to be made

Reply:
@vorot93 could we cut a new release good sir?

Reply:
https://crates.io/crates/fastrlp/0.1.2

Reply:
🚀🚀🚀 thanks! bumped to `0.1.2`